pandas-stubs/core/indexes/timedeltas.pyi

```diff
@@ -36,7 +36,7 @@ class TimedeltaIndex(DatetimeTimedeltaMixin, TimedeltaIndexProperties):
     def get_value(self, series, key): ...
     def get_loc(self, key, tolerance=...): ...
     def searchsorted(self, value, side: str = ..., sorter=...): ...
-    def is_type_compatible(self, typ) -> bool: ...
+    def is_type_compatible(self, kind: str) -> bool: ...
```

Comment:
Maybe only fix this in `indexes/base.pyi` and not declare it here.  Not sure why we have it here when it is in the base class.